random-book/src/main.tsx

```diff
@@ -0,0 +1,64 @@
+// import * as React from "react";
+// import * as ReactDOM from "react-dom";
+
+// const appDiv = document.getElementById("app");
+
+// if (!(appDiv instanceof HTMLDivElement)) {
+// 	throw new Error("No div with id 'app' found");
+// }
+
+// const App = () => {
+// 	const [counter, setCounter] = React.useState(0);
+
+// 	return (
+// 		<main>
+// 			<p>{counter}</p>
+// 			<button onClick={() => setCounter(counter + 1)}>
+// 				Increment
+// 			</button>
+// 		</main>
+// 	);
+// };
+
+// ReactDOM.render(<App />, appDiv);
+
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+const appDiv = document.getElementById("app");
+
+if (!(appDiv instanceof HTMLDivElement)) {
+	throw new Error("No div with id 'app' found");
+}
+
+function RandomTitle(App) {
+    const [title, newTitle] = React.useState("Click For New Title");
+
+    const { titles } = App;
+    return (
+        <main>
+            <p>{title}</p>
```

Comment:
Should also display the book with a random background-color 😉

Reply:
i will do it after doing second exercise.

Reply:
👍